os_stub/memlib/copy_mem.c

```diff
@@ -9,35 +9,65 @@
  **/
 
 #include "base.h"
+#include "library/debuglib.h"
 
 /**
- * Copies a source buffer to a destination buffer, and returns the destination buffer.
+ * Copies bytes from a source buffer to a destination buffer.
  *
```

Comment:
I recommend we add description on the difference between this API and C11 memcpy_s().
For example, C11 requires to fill dst_buf with 0, in case of runtime violation. But it is not in this API.

Reply:
Agree.